src/main/java/hw3/components/AbstractComponent.java

```diff
@@ -0,0 +1,17 @@
+package hw3.components;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class AbstractComponent {
+
+  protected WebDriver driver;
+  protected WebDriverWait wait;
+
+  public AbstractComponent(WebDriver driver) {
```

Comment:
Why do you set constructor as `public`

Reply:
fixed. it is more correct to do with protected constructor